pkgdb/src/buildenv/assets/etc/profile.d/0500_python.sh

```diff
@@ -20,7 +20,7 @@ print( "{}.{}".format( sys.version_info[0], sys.version_info[1] ) )'
 fi
 
 # Only run if `pip' is in `PATH'
-if command -v pip > /dev/null 2> /dev/null; then
+if [[ -x "$FLOX_ENV/bin/pip3" ]]; then
   export PIP_CONFIG_FILE=$(@realpath@ --no-symlinks $FLOX_ENV/../../pip.ini)
```

Comment:
writing to that pip.ini file seems rather dangerous, considering that it will _replace_ the existing file if any.
Not sure if we should to fix this here but i can open an issue to track this at least.

Reply:
ah pip.init is written in the .flox dir